clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -600,6 +601,30 @@ public void testCoordinatorNotAvailable() {
         assertTrue(coordinator.coordinatorUnknown());
     }
 
+    @Test
+    public void testSyncCommitWithoutOffsetsAndPendingAsyncCommit() {
```

Comment:
maybe `testEnsureCompletingAsyncCommitsWhenSyncCommitWithoutOffsets` ? I thought the original naming sounds a bit like firing sync commit without offset and async commit.

Reply:
Sorry, somehow I missed this comment. Yes, I agree your suggestion is much better. Applied in https://github.com/apache/kafka/pull/13678/commits/13bccc7afe6422f48f87bda031083dde50802649